concent_api/core/subtask_helpers.py

```diff
@@ -112,14 +114,15 @@ def update_timed_out_subtasks(
 def update_subtask_state(
     subtask,
     state,
+    next_deadline: Optional[int] = None
 ):
     logging.log_change_subtask_state_name(
         logger,
         subtask.state,
         state,
     )
-    subtask.state    = state
-    subtask.next_deadline = None
+    subtask.state = state
+    subtask.next_deadline = next_deadline if next_deadline is None else parse_timestamp_to_utc_datetime(next_deadline)
```

Comment:
I think that `None if next_deadline is None else parse_timestamp_to_utc_datetime(next_deadline)` would be more readable

Reply:
done